openfermioncirq/trotter/simulate_trotter_test.py

```diff
@@ -35,40 +33,49 @@ def fidelity(state1, state2):
     return abs(numpy.dot(state1, numpy.conjugate(state2)))**2
 
 
+def load_molecular_hamiltonian(
```

Comment:
why is this function in OpenFermion-Cirq. It should definitely not be in this library, let alone this module. Please remove.

Reply:
It's used to load test Hamiltonians (see below). I'm happy to put it somewhere in OpenFermion that you think is appropriate.

Reply:
yes I think that would be more appropriate. Perhaps in the "testing" module.